### DISCUSSION TITLE
Labels for issues

### DISCUSSION THREAD

**Discussion by xcorail** [ossf/wg-best-practices-os-developers#91] [Ideas]
The group decided that it would be good to have labels in our issues to categorize, to plan, and to prepare the agenda. Here is a proposal: 

* **categories** corresponding to our work streams: `Scorecards` `Fundamentals-course` `SKF` …
* **milestones**: 
  * `immediate` (something we need to do NOW, such as collaborating on an announcement with a deadline) `backlog` (needs to be discussed and assigned / planned) `nice-to-have` (we won’t discuss it in the short future, but we think we need to keep it :man-shrugging:) + specific labels for identified milestones such as `OSSNA-2023` or `2022-Q4` … 
  * So basically the lifecycle of an issue is `<no-label>` at creation. The group should regularly triage all the issues without a date label, then after a first triage it goes to one of `immediate`/`backlog`/`nice-to-have`. The `immediate`s are treated, the `backlog` ones are discussed, and planned, and when they are planned they inherit a specific label such as Q4 or a specific event. 
* `next-meeting`: to generate automatically the agenda / to make everyone interested in the issue aware that they might want to attend next meeting
* We can if needed prefix the labels with identifiers such as `cat-` or `M-` if we need for example a way to identify easily the uncategorized or the unplanned issues. We don't necessarily need to overthink it as labels can be changed later.

The next step is to discuss this proposal in a group meeting.

**Comment by torgo** [comment]
Looks good to me! In @w3ctag (specifically in https://github.com/w3ctag/design-reviews/issues) we use milestones as well as labels - we may want to consider this here. We also automatically add an `untriaged` label to new issues. I'd also suggest we need an issue template which guides creation of new issues to include some the info we'd like to see in new issues.

**Reply by xcorail** [reply]
Thanks for the feedback. Indeed, once an issue is assigned and planned, instead of using a label for the planning target, we could use a milestone instead, that makes more sense.

+1 for the automated `untriaged` label at issue creation, and also agree with the issue template(s)
